arbiter-core/src/environment/cheatcodes.rs

```diff
@@ -36,6 +38,28 @@ pub enum Cheatcodes {
         /// The value to overwrite the storage slot with.
         value: ethers::types::H256,
     },
+    /// Fetches the `DbAccount` account at the given address.
+    Access {
+        /// The address of the account to fetch.
+        address: ethers::types::Address,
+    },
+}
+
+/// Wrapper around [`AccountState`] that can be serialized and deserialized.
+#[derive(Debug, Clone, Default, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+pub enum AccountStateSerializable {
```

Comment:
Consider PRing this change into `revm`? https://github.com/bluealloy/revm/blob/310cabaa0a1306687dd4a8218a74c27fb338bbc8/crates/revm/src/db/in_memory_db.rs#L333

Reply:
Actually, I don't quite see why we need the serde traits now as I don't think they're used anywhere. Do you have plans to use them elsewhere?